package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "peggy",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
don't touch the version until the release process please.

Reply:
ok

Reply:
sorry, did this by habit.  i just do it when i start work so i don't forget later

i'll skip that on peggy

Reply:
If that's a thing we're interested in, let's make it part of the release process to change the version to x.x.xRC or something right after we tag a release.  But let's make that a separate issue and PR.

Reply:
i'm happy to roll back having touched the specific version, it's just a habit.  this isn't important to me

the part i'm actually involved in is making sure the machinery is maintained, so that when it comes time for *you* to change the version, the process itself is automated and reliable

Reply:
Version is rolled back